### DISCUSSION TITLE
r/city-china-zhejiang-shaoxing-xuehuatan

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#21] [Announcements]
# r/city-china-zhejiang-shaoxing-xuehuatan

游记 | 酷热的夏天，发现了玩儿水的秘境

https://blog.ops-coffee.cn/r/city-china-zhejiang-shaoxing-xuehuatan

**Comment by HogwartsRico** [comment]
哈哈，这次来我家乡了。

**Reply by ops-coffee** [reply]
你的家乡好漂亮

**Comment by woshizhd** [comment]
四明山附近是个好地方😄。绍兴工厂多，但是一进山环境就不同了哈哈哈😃路就在两山夹缝之间，开车看风景是真的好😁
章镇附近有个“樟树下民宿”，我住过两次，旁边的小溪可以钓鱼🎣让老板娘加工一下可鲜美了😁。老板娘种了好多果树，现在应该有西瓜🍉再过一个月板栗🌰就可以吃了，章镇路边的板栗真是超多，随处可见，第一次见我大受震惊。
“樟树下民宿”属于岭南乡，乡政府的街上（距离1.5km左右）有家“斌元馒头店”，上午去可以买到老面肉包子，可好吃了😋

**Reply by ops-coffee** [reply]
这个分享很棒，应该能帮助很多人
